configs/platforms/osx-13-x86_64.rb

```diff
@@ -0,0 +1,8 @@
+platform 'osx-13-x86_64' do |plat|
+  plat.inherit_from_default
```

Comment:
Since we're inheriting from default for this platform, we'll need to add it upstream in Vanagon before merging this PR. It looks like this work is ticketed out in PA-5326

Reply:
Yess, I have made changes there too. Will raise a PR for the same too.

Reply:
@mhashizume  please review this too : https://github.com/puppetlabs/vanagon/pull/807
Thanks!